tests/Model/EditableFormField/EditableFileFieldTest.php

```diff
@@ -72,4 +73,40 @@ public function testAllowEmptyTitle()
         $field->Name = 'EditableFormField_123456';
         $this->assertEmpty($field->getFormField()->Title());
     }
+
+    public function testOnBeforeWrite()
+    {
+        $this->logOut();
+
+        /** @var EditableFileField $fileField */
+        $fileField = $this->objFromFixture(EditableFileField::class, 'file-field');
+
+        $defaultFolder = Folder::find('Form-submissions');
+        $this->assertNotEmpty($defaultFolder, 'Default Folder was created along with the EditableFileField');
+        $this->assertFalse($defaultFolder->canView(), 'Default Folder default to being restricted');
+        $this->assertFalse((boolean)$fileField->FolderConfirmed, 'EditableFileField are not Folder Confirmed initially');
```

Comment:
This one is a tiny bit more subtle. The way the onBeforeWrite logic is currently written, I don't think it's possible to create an EditableFileField that is not `FolderConfirmed`